docs/next.config.js

```diff
@@ -1,9 +1,8 @@
 const { withSentryConfig } = require("@sentry/nextjs");
 const withNextra = require("nextra")({
   theme: "nextra-theme-docs",
-  themeConfig: "./theme.config.js",
-  unstable_flexsearch: true,
-  unstable_staticImage: true,
+  themeConfig: "./theme.config.tsx",
+  defaultShowCopyCode: true,
```

Comment:
I don't know why this isn't set by default, have to check on that one (cc @shuding)

Reply:
Will make that the default!